builder_test.go

```diff
@@ -121,8 +121,8 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 						},
 					},
 				}
-				bpA := testmock.NewMockBuildpack(mockCtrl)
-				bpB := testmock.NewMockBuildpack(mockCtrl)
+				bpA := testmock.NewMockBuildpack(mockCtrl, builder.Group.Group[0].API)
```

Comment:
To force `bpA` to return a specific value for `BuildpackAPI` you can do `bpA. EXPECT().BuildpackAPI().Return(my-value)`

Reply:
But I think if we follow https://github.com/buildpacks/lifecycle/pull/520/files#r568666640 we won't need it...

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/520/commits/63f4e9a470ff92f133caa5ffff56d047d6fa3295.